src/mlpack/core/optimizers/lbfgs/test_functions.hpp

```diff
@@ -0,0 +1,175 @@
+/**
+ * @file test_functions.hpp
```

Comment:
This function is now in the problems directory.

Reply:
Oops, I think this was an accidental change during the merge process.  I've removed the files.